notification-hubs-sdk/build.gradle

```diff
@@ -41,6 +41,8 @@ android {
 
     testOptions {
         execution 'ANDROIDX_TEST_ORCHESTRATOR'
+        unitTests.returnDefaultValues = true
+        unitTests.includeAndroidResources = true
```

Comment:
This must be why you were able to include the tests I mention below in the `test` source set instead of the `androidTest` source set. Let's get rid of this line, and move the tests over.

Reply:
Answered above(robolectric thread)